CMakeLists.txt

```diff
@@ -160,6 +162,11 @@ foreach (CONFIG ${CMAKE_BUILD_TYPE} ${CMAKE_CONFIGURATION_TYPES})
     string(REGEX REPLACE "-O[^ ]*" "" CMAKE_CXX_FLAGS_${CONFIG} "${CMAKE_CXX_FLAGS_${CONFIG}}")
 endforeach ()
 
+if (ARCH_LOONGARCH64)
```

Comment:
these do not belong here, but in their respective cmake file. Please, see the other architectures.

Reply:
Done.